utils/src/main/scala/za/co/absa/hermes/utils/SparkCompatibility.scala

```diff
@@ -21,7 +21,7 @@ import za.co.absa.commons.version.impl.SemVer20Impl.SemanticVersion
 
 object SparkCompatibility {
   val minSparkVersionIncluded: SemanticVersion = semver"2.4.2"
-  val maxSparkVersionExcluded: SemanticVersion = semver"3.2.0"
+  val maxSparkVersionExcluded: SemanticVersion = semver"3.2.3"
```

Comment:
I would put 3.3.0 here - it makes sense that when working with 3.2.2, it would work with higher patch versions, too.

Originally, the logic, I belive, was the same.

Reply:
Wasnt there a PR removing the upper bound?

Reply:
Upper bound increased to 3.3.0 in 0da1a40805f0860d784ee4133fdbd2f57b51ff07.

Reply:
> Wasnt there a PR removing the upper bound?
I don't remember that, but upper bound is usually optional anyway.